### PR TITLE
Bfi ui

### PR DESCRIPTION
**Pull request by xagau**

### CODE REVIEW

File: src/init.cpp [hunk 1]
```diff
@@ -359,14 +359,17 @@ void SetupServerArgs()
         "-dbcrashratio", "-forcecompactdb",
         // GUI args. These will be overwritten by SetupUIArgs for the GUI
         "-choosedatadir", "-lang=<lang>", "-min", "-resetguisettings", "-splash", "-uiplatform"};
-
+	// VBK
+	gArgs.AddArg("-bfiendpoint", "Default end point for BFI setup.", ArgsManager::ALLOW_STRING, OptionsCategory::OPTIONS);
```
Comment (Warchant) [inline src/init.cpp:363]: could you pls setup your editor to use spaces instead of tabs?

Comment (xagau) [reply src/init.cpp:363]: Changed for future reference.

File: src/qt/optionsdialog.cpp [hunk 2]
```diff
@@ -185,6 +186,13 @@ void OptionsDialog::setModel(OptionsModel *_model)
     /* Display */
     connect(ui->lang, static_cast<void (QValueComboBox::*)()>(&QValueComboBox::valueChanged), [this]{ showRestartWarning(); });
     connect(ui->thirdPartyTxUrls, &QLineEdit::textChanged, [this]{ showRestartWarning(); });
+
+
+	/* VBK BFI */
+	//connect(ui->bfiDataSourceUrls, &QLineEdit::textChanged, this, &OptionsDialog::bfiChanged );
+	/* VBK BFI */
```
Comment (Warchant) [inline src/qt/optionsdialog.cpp]: is this PR a work-in-progress?

Comment (xagau) [reply src/qt/optionsdialog.cpp]: No.
That is a reminder to me if we later add the capability to write conf files.

Comment (xagau) [reply src/qt/optionsdialog.cpp]: Removed future proofing code from production release.

**Pull request merged**